atomate/vasp/workflows/base/neb.py

```diff
@@ -0,0 +1,359 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""
+This module defines the Climbing Image Nudged Elastic Band (CI-NEB) workflow.
+"""
+
+import os
+from datetime import datetime
+
+from pymatgen.core import Structure
+from pymatgen_diffusion.neb.io import get_endpoints_from_index
+from fireworks.core.firework import Workflow
+
+from atomate.vasp.fireworks.core import NEBFW, NEBRelaxationFW
+from atomate.utils.utils import get_logger
+
+__author__ = "Hanmei Tang, Iek-Heng Chu"
+__email__ = 'hat003@eng.ucsd.edu, ihchu@eng.ucsd.edu'
+
+logger = get_logger(__name__)
+spec_orig = {"wf_name": "CINEB",
```

Comment:
Yes, this part was not good enough. Now it is renamed as "default_spec" and used to pass certain parameters like "sort_tol" and "interpolation_type", which are related to the construction of initial NEB migration path. As the actual NEB calculations often require multiple nodes, we still need to update the number of requested nodes in spec (via "_queueadapter"). To test the performance of the current NEB workflow, we have carried out short-time tests on both SLURM and PBS queueing systems. The results are as what we expect.